scripts/setup_consensus_schema.py

```
@@ -0,0 +1,249 @@
+        tables = cursor.fetchall()
+        logger.info(f"Tables created: {len(tables)}")
+        for table in tables:
+            logger.info(f"  ✓ {table[0]}")
+
+        cursor.close()
+        conn.close()
+
+        logger.info("✅ Schema setup completed successfully!")
+        return True
+
+    except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)